.github/CODEOWNERS

```
@@ -208,7 +208,11 @@
 /packages/synthetics @elastic/uptime
 /packages/sysmon_linux @elastic/security-external-integrations
 /packages/system @elastic/obs-infraobs-integrations
+/packages/system/changelog.yml @elastic/obs-infraobs-integrations @elastic/security-external-integrations
+/packages/system/data_stream/auth @elastic/security-external-integrations
+/packages/system/data_stream/security @elastic/security-external-integrations
 /packages/system/kibana @elastic/elastic-agent-data-plane @elastic/kibana-visualizations
```

Should @elastic/elastic-agent-data-plane be replaced here with @elastic/obs-infraobs-integrations?
```suggestion
/packages/system/kibana @elastic/obs-infraobs-integrations @elastic/kibana-visualizations
```

---

Seems fine to me.. @andresrc @rameshelastic for any thoughts?

---

LGTM